projects/demo/src/modules/app/home/home.template.html

```diff
@@ -111,6 +111,17 @@
                         <code>npm i @taiga-ui/styles</code>
                         and put this code:
 
+                        <tui-notification
+                            status="warning"
+                            class="tui-space_top-3"
+                        >
+                            [WARNING]: In version 3.0 we removed support normalize(v7) out-of-the-box
```

Comment:
nitpicking:

```suggestion
                            In version 3.0 we removed support normalize(v7) out-of-the-box
```

It is already `status="warning"`)